docs/source/faq/run_dps_job.ipynb

```diff
@@ -49,16 +49,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [],
    "source": [
+    "# Add some search results to create a list of granules\n",
+    "granules = []\n",
     "mit_result = maap.submitJob(\n",
```

Comment:
Should we update this to run on GCC?

Reply:
that's a great idea but I think this is just meant to be an example for now. I'm hesitant to run "real" jobs until we discuss with JPL team about that.